app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -178,6 +183,13 @@ public void onSetSpeedAbilityChanged() {
         };
     }
 
+    static protected TextView getTxtvFFFromActivity(MediaplayerActivity activity) {
```

Comment:
can you change this to ``protected static``? This order is really unconventional